corehq/util/quickcache.py

```diff
@@ -1,8 +1,11 @@
 from __future__ import absolute_import
 from __future__ import unicode_literals
 import warnings
+import hashlib
```

Comment:
nit: these imports are out of order

Reply:
huh. I didn't even know we had a convention of sorting imports. What's the convention? Just one-word imports like this get sorted? All get sorted get sorted? builtin vs. third party vs. same-repo imports each get sorted separately from each other?

Reply:
We don't have our own convention. [PEP8](https://www.python.org/dev/peps/pep-0008/#imports) says:

> Imports should be grouped in the following order:
>
> standard library imports
> related third party imports
> local application/library specific imports
>
> You should put a blank line between each group of imports.

I thought it also said something about those being alphabetical, but it looks like that's from a flake8 plugin I use, not officially pep8

In this case I think it would go:

```
future imports

stdlib imports

celery/quickcache

corehq
```

which is what it is, but without the spacing

Reply:
Okay, yeah that sounds right, and I've never really seen us be consistent about it. Do you know if that's something we can turn on for stickler? I think that's the type of thing where we either enforce it automatically or we don't even try

Reply:
it doesn't look like there's a check in `pep8` or `pylint` that will do it which makes sense as there isn't a general way to say what's a project import vs 3rd party import. the alphabetical check I have locally is from a `flake8` plugin called `hacking`, but I don't you can add plugins to stickler

> I think that's the type of thing where we either enforce it automatically or we don't even try

Eh. Seems like a pretty easy convention to follow that slightly improves readability/consistency so I'll probably continue commenting as a nitpick without blocking on it. and change imports as I see them.